packages/controller-utils/src/create-service-policy.ts

```diff
@@ -110,17 +122,19 @@ export function createServicePolicy({
     // do nothing
   },
 }: {
+  maxRetries?: number;
```

Comment:
Should we call this `retries` instead of `maxRetries`? How would you expect to use this function?

Reply:
I would opt in to keep `maxRetries` for clarity and to avoid ambiguity.